privacy-protections/click-to-load/index.html

```diff
@@ -1,62 +1,59 @@
-<br /><div class="fb-save" data-uri="https://developers.facebook.com/docs/plugins/1" data-size="large"></div>
-<br /><div class="fb-share-button" data-href="https://developers.facebook.com/docs/plugins/" data-layout="button_count" data-size="small"><a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&amp;src=sdkpreparse" class="fb-xfbml-parse-ignore">Share</a></div>
-
-<h2> All the social buttons in iFrames </h2>
-<iframe src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins2%2F&width=450&layout=standard&action=like&size=small&share=true&height=35&appId" width="450" height="35" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
-<br /><iframe src="https://www.facebook.com/v2.0/plugins/like.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins1%2F&width=450&layout=standard&action=like&size=small&share=true&height=35&appId" width="450" height="35" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
-<br /><iframe src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&layout=button_count&size=small&width=121&height=20&appId" width="121" height="20" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
-
-<h2> Login buttons (SDK & custom) </h2>
+<!-- Load the facebook SDK in the recommended way -->
+<div id="fb-root"></div>
+<script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v14.0" nonce="1SgoaWbh"></script>
+<p><a href="../../">[Home]</a>↣<a href="../">[Privacy Protections Tests]</a>↣<strong>[Facebook click to load]</strong></p>
+
+<h2>About ClickToLoad Tests</h2>
+<p>This page embeds the Facebook SDK, and several Facebook iFrames. It also attempts to create
+a variety of social elements using the SDK.</p>
+
+<p>The page will count how many Facebook requests have been sent. In some cases, the browser may start requesting iFrame links, which should be blocked by the extension (but will be counted in the Facebook calls). Calls should not exceed iFrames (and manual verification that those network calls were blocked is a good practice). Once elements are clicked, the count should increase and elements should display (in most cases - this page does not use a valid SDK APP ID, so not all functions work correctly, login in particular.).</p>
+
+<h2>Metrics</h2>
```

Comment:
@ladamski What do you think about:
1. removing the metrics section and the corresponding JS code
2. replacing the second stanza of the introduction with an explanation of what should happen when CTL is enabled (similar to how we do for [the YouTube CTL test page](https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/)

My thinking is that the metrics aren't reliable, they seem effectively random and can't be trusted to verify the feature is working. They add noise to the page and complexity to the code, making both harder to understand. Also, they aren't required for the integration tests anyway.

I figure that having just a few notes to explain what the feature is expected to do (e.g. replace most FB with content, show a warning prompt when login buttons are clicked, etc) would be much more useful for folks trying to use the test page.

Reply:
Agree that is not super clear, the iframe count doesn't really depend on protection status anyway.  The "Facebook calls" one though should be zero when protections are enabled, so I think it should be replaced with a simpler boolean metric that remains "BLOCKED" unless a call is detected.

I like the idea of adding an intro paragraph, that would be very useful!

Reply:
This is a huge improvement, nice one! :1st_place_medal:

Reply:
Optional: Perhaps rename "Facebook Resource Loads" to something like "Automatic Facebook Resource Loads"? Doesn't really roll off the tongue, but I'm thinking that it might be kind of misleading to keep saying "NONE" otherwise after the user clicked to unblock some Facebook content?